src/raw/mod.rs

```diff
@@ -1,4 +1,5 @@
 #![allow(unsafe_code)]
+#![cfg_attr(feature = "cargo-clippy", deny(clippy::all))]
```

Comment:
As mentioned on the .travis.yml, I believe this is better done on the command line. Furthermore, this would deny them only inside this module and submodules, which is weaker than what was previously.

Reply:
Before I moved this here, the build wasn't being denied at all based on a lint failure.

Reply:
Appears to work now. I tried pushing a build with a failing lint